apps/judicial-system/web/src/routes/Admin/UserForm/UserForm.tsx

```diff
@@ -216,7 +219,8 @@ export const UserForm: FC<Props> = ({
             required
           />
         </Box>
-        {user.institution?.type === InstitutionType.PROSECUTORS_OFFICE ? (
+        {user.institution?.type &&
+        prosecutorsOfficeTypes.includes(user.institution.type) ? (
```

Comment:
Nit: extract as type check function in `@island.is/judicial-system/types`? So we could call `isProsecutorOfficeType(type)`

Reply:
Good idea